README.org

```diff
@@ -147,6 +147,16 @@ The log message will be collected in a buffer =*overleaf-[document-id]*=.
 
 Feel free to open an issue providing this log.
 
+** Customizing WebDriver behaviour
+Customize ~overleaf-webdriver-init-function~ to pass [[https://developer.mozilla.org/en-US/docs/Web/WebDriver/Reference/Capabilities/firefoxOptions][options]] (like the path to the Firefox binary, what profile should be used, etc) to ~geckodriver~.
+For example, this code snippet could be helpful for users with a non-standard Firefox installation (i.e. not at =/Applications/Firefox.app/Contents/MacOS/firefox-bin= for macOS):
+#+begin_src elisp
+(defun my/overleaf--webdriver-make-session ()
+  (let ((caps (make-instance 'webdriver-capabilities-firefox)))
+    (webdriver-capabilities-add caps :binary (expand-file-name "~/Applications/Home Manager Apps/Firefox Nightly.app/Contents/MacOS/firefox-bin") t)
+    (make-instance 'webdriver-session :requested-capabilities caps)))
+#+end_src
```

Comment:
I think the example would be more complete if it had `(setq overleaf-webdriver-init-function #'my/overleaf--webdriver-make-session)` at the end.

The double dash usually denotes a private function, which seems a bit strange to me here.